src/components/header/Alarm/index.tsx

```diff
@@ -27,9 +25,9 @@ const Alarm = ({ isAlarmExisted, isActivated, onClick, alarmRef }: AlarmProps) =
       <div className={cx('dot', 'dot-left', { 'dot-activated': isActivated })}></div>
       <div className={cx('frame-inner', { 'frame-inner-activated': isActivated })}>
         {isAlarmExisted ? (
-          <Image src={full.url} alt={full.alt} width={20} height={20} />
+          <img src={full.url} alt={full.alt} width={20} height={20} />
         ) : (
-          <Image src={empty.url} alt={empty.alt} width={20} height={20} />
+          <img src={empty.url} alt={empty.alt} width={20} height={20} />
```

Comment:
```suggestion
<img src={isAlarmExisted ? full.url : empty.url ... />
```
이런 식으로 사용하면 코드 가독성이 더 좋아질 것 같습니다~